openmcyclus/DepleteReactor.py

```diff
@@ -347,97 +420,140 @@ def get_material_bids(self, requests): # phase 2
+
             else:
                 mats = []
-            print("time:", self.context.time, "mats to trade matching request commod:", mats)
+            print(
+                "time:",
+                self.context.time,
+                "mats to trade matching request commod:",
+                mats)
             if len(mats) == 0:
-                continue 
+                continue
+
+            recipe_comp = self.context.get_recipe(
+                self.fuel_outrecipes[commod_index])
 
-            recipe_comp = self.context.get_recipe(self.fuel_outrecipes[commod_index])
-    
             for req in reqs:
                 tot_bid = 0
                 for jj in range(len(mats)):
                     tot_bid += mats[jj].quantity
                     qty = min(req.target.quantity, self.assem_size)
                     mat = ts.Material.create_untracked(qty, recipe_comp)
                     for kk in range(self.spent_fuel.count):
-                        bids.append({'request':req,'offer':mat})
+                        bids.append({'request': req, 'offer': mat})
                     if tot_bid >= req.target.quantity:
                         break
             tot_qty = 0
             for mat in mats:
                 tot_qty += mat.quantity
         if len(bids) == 0:
-            return 
+            return
 
-        port = {'bids':bids}
+        port = {'bids': bids}
         print("time:", self.context.time, "portfolio:", port)
         print("time:", self.context.time, "respond", len(bids), "assemblies")
         return port
 
-    def get_material_trades(self, trades): #phase 5.1
+    def get_material_trades(self, trades):  # phase 5.1
         '''
         Trade away material in the spent_fuel material buffer.
 
         Pull out the material from spent fuel inventory
-        For each trade, get the commodity name, get the 
+        For each trade, get the commodity name, get the
         composition of the trade.
 
-        Then trade the materials from the spent fuel inventory. 
+        Then trade the materials from the spent fuel inventory.
+
+        Parameters:
+        -----------
+        trades: tuple
+            tuple of material objects requested matched to the 
+            material responses
+
+        Returns:
+        --------
+        responses: dict
+            dictionary of {Material object:Material object}. The 
+            key is the Material request being matched. The value is the 
+            Material in the spent fuel inventory
         '''
         responses = {}
         mats = self.pop_spent()
-        print("time:", self.context.time, "trade away", len(trades), "assemblies")
+        print(
+            "time:",
+            self.context.time,
+            "trade away",
+            len(trades),
+            "assemblies")
+        print(trades)
         for ii in range(len(trades)):
             print("time:", self.context.time, "number of trades:", len(trades))
+            print(trades[ii])
```

Comment:
Consider adding more detail (such as `print(f'Trade {ii}: {trades[ii]}')` or something similar) so the information is easier to read